integration_test/third_party_apps_data/applications/iis/enable

```diff
@@ -1,4 +1,14 @@
 Add-Content 'C:\Program Files\Google\Cloud Operations\Ops Agent\config\config.yaml' "
+metrics:
+  receivers:
+    iis_v2:
+      type: iis
+      receiver_version: '2'
+  service:
+    pipelines:
+      iispipeline:
+        receivers:
+          - iis_v2
```

Comment:
See the commentary on the MSSQL PR since it directly applies here as well: https://github.com/GoogleCloudPlatform/ops-agent/pull/554/files?file-filters%5B%5D=.go&file-filters%5B%5D=.lua&file-filters%5B%5D=.md&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension&show-viewed-files=true#r859050298

Reply:
So, from my understanding this is still testing both configs because if you add the `_v2` suffix it won't override the existing IIS receiver that's built in. See the [test case](https://github.com/GoogleCloudPlatform/ops-agent/pull/534/files#diff-546c7159fc951077f2d75715a3f8a369ff1c8fd5372ce93d5eac87564bcb7fdeR15) and generated otel config for the test case I wrote were the [old iis metrics ](https://github.com/GoogleCloudPlatform/ops-agent/pull/534/files#diff-5e935c88be9432e5dd4b684f5a9acf1049b03467c463202ad5958afc97813eefR483) exist along side the[ new receiver](https://github.com/GoogleCloudPlatform/ops-agent/pull/534/files#diff-5e935c88be9432e5dd4b684f5a9acf1049b03467c463202ad5958afc97813eefR464)

Reply:
You're right, I missed that part. It might be OK then. We can continue the discussion on the MSSQL thread.